src/ai/PathFinder.cpp

```diff
@@ -151,29 +151,25 @@ class PathFinder::OpenNodeList {
 
 class PathFinder::ClosedNodeList {
 	
-	typedef std::vector<Node*> NodeList;
-	NodeList nodes;
+	typedef std::vector<bool> NodeList;
+	NodeList *nodes;
 	
 public:
+
+	ClosedNodeList(int nodeCount) {
+		nodes = new NodeList(nodeCount, false);
```

Comment:
I know I suggested using an array, but have you checked how large these get? Probably not a problem, but this does add a minimum overhead to even small pathfinding operations. Then again, the pathfinder is normally not used for short paths (when there is a direct line of sight).